openshift_metrics/utils.py

```diff
@@ -75,24 +103,32 @@ def query_metric(openshift_url, token, metric, report_start_date, report_end_dat
     return data
 
 
-def get_namespace_annotations():
+def get_namespace_attributes():
     """
-    Returns namespace annotations
-    Used for finding coldfront pi name and id
+    Returns allocation attributes from coldfront associated
+    with all projects/namespaces.
+
+    Used for finding coldfront PI name and institution ID.
     """
-    token = os.environ.get("OPENSHIFT_TOKEN")
-    api_url = os.environ.get("OPENSHIFT_API_URL")
+    client = ColdFrontClient(
+        "https://keycloak.mss.mghpcc.org",
+        os.environ.get("CLIENT_ID"),
+        os.environ.get("CLIENT_SECRET")
+    )
 
-    if token is not None:
-        openshift.set_default_token(token)
-    if api_url is not None:
-        openshift.set_default_api_url(api_url)
+    coldfront_url = os.environ.get("COLDFRONT_URL",
+        "https://coldfront.mss.mghpcc.org/api/allocations")
```

Comment:
This will not return allocations that are not active. [Need to pass](https://github.com/nerc-project/coldfront-plugin-api/blob/7f5bad4e90a6718fb3c2e76efdd29d07da87f588/src/coldfront_plugin_api/urls.py#L20-L21) `?all=true` to get expired/denied/revoked/etc.